src/components/ModifiersGroup.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div>
+    <div class="title-modify">{{groupTitle}}</div>
+    <q-list inset-separator class="q-pa-none listing-modifier">
+      <q-item v-for="(modifier,index) in data" :key="index">
+        <q-item-main class="text-secondary">
+          <q-radio class="fit" v-if="groupType==='radio'" v-model="modifiers" :val="modifier" :label="modifier.name" color="primary"/>
```

Comment:
no need to use watch to get defaultModifiers , just set defaultModifiers inside mounted event